### DISCUSSION TITLE
Access to directory

### DISCUSSION THREAD

**Discussion by hopfrogsalerno** [AzuraCast/AzuraCast#4366] [Q&A]
How do I access this directory with FTP? 
/var/azuracast/stations/radio_example/recordings

In /var/azuracast I find only the files . env and docker.
Thanks

**Comment by Vaalyn** [comment]
The path `/var/azuracast/stations/radio_example/recordings` is located in the containers isolated filesystem.

You can find the station directories volume on the host at `/var/lib/docker/volumes/azuracast_station_data/_data/`

**Reply by hopfrogsalerno** [reply]
OK, thank you. I’ve never used Docker and I don’t know it... I’ll have to study it

**Reply by phoenixqss** [reply]
> Путь `/var/azuracast/stations/radio_example/recordings`находится в изолированной файловой системе контейнеров.
> 
> Вы можете найти том каталогов станций на хосте по адресу `/var/lib/docker/volumes/azuracast_station_data/_data/`

А где я могу найти все изображения и главную картинку сайта для того что бы заменить на свои? Подскажите мне пожалуйста .
